### PR TITLE
New section "Iterating Over Compounds" to Assembly documentation.

### PR DESCRIPTION
**Pull request by slobberingant**
Iterating over compounds like this has been a solution to a few of my previous problems and I haven't seen it in the documentation. Hope it helps.

### CODE REVIEW

**Review by gumyr — CHANGES_REQUESTED** [review]
One minor change if you don't mind.  The `sum` doesn't need the list comprehension, if its implemented as `sum(part.volume for part in assembly3.solids())` it is called a **generator expression** and is the recommended way to do sums like this.

File: docs/assemblies.rst [hunk 1]
```diff
@@ -160,6 +160,28 @@ adds the following attributes to :class:`~topology.Shape`:
 
     .. _pack:
 
+************************
+Iterating Over Compounds
+************************
+
+As Compounds are containers for shapes, Build123 can iterate over these as required.
```
Comment (jdegenstein) [inline docs/assemblies.rst]: please use "build123d" -- all lowercase as per convention in Python

**Comment by slobberingant** [comment]
Feedback has been incorporated. Thank you.

**Comment by gumyr** [comment]
Thanks!

**Pull request merged**